docs/user_manual/site_configuration.md

```diff
@@ -190,3 +190,11 @@ easier to find that they are related because they are connected to the same ghos
 
 {% include userman-toplink.md %}
 
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Strict Index Checking" %}
+
+By default, any Library Aliquot can be added to a Pool. When Strict Index Checking is enabled, Library Aliquots
+cannot be added to a Pool when their addition would cause duplicate or near-duplicate indices to exist within the 
+Pool. 
```

Comment:
Entity types (library aliquots, pools, etc.) and option names (strict index checking) shouldn't be capitalized in the user manual except in titles

Reply:
done